### ISSUE TITLE
Add Name on Hover to Country Icons

### ISSUE THREAD

**Issue by alexmaasswca** [thewca/worldcubeassociation.org#6066]
**Is your feature request related to a problem? Please describe.**
I'm frustrated by the names of countries being replaced with flags on results pages, I know soon a fix should be implemented where you can hover over them, but there are situations where countries have super similar flags, if it can either be changed back to the names of countries, or have both the flag and the names of countries, that would be much better, it is absolutely awful when for example, you're displaying New Zealand and Australian people together, the flags are almost identical 
It is obviously the better way to display the country, as you don't have to hover over every single country flag to get the name of the country.
![image](https://user-images.githubusercontent.com/48773372/109441050-2c7f5e00-7a02-11eb-8119-1f858b5dae40.png)

**Describe the solution you'd like**
Include the names of the countries next to the flags, or replace the flags with the names of the country. 
**Additional context**
https://forum.worldcubeassociation.org/t/can-we-please-revert-or-fix-the-issues-with-how-competition-results-are-displayed/11843?u=alex_maass

**Comment by denthebro** [comment]
I agree with this change. This could also help with accessibility for some colorblind people.

**Comment by Jambrose777** [comment]
I think hovering over the flag to find out its country is completely reasonable and doesnt take up any precious space on a website. This is currently implemented on the Competition's Results that Alex posted, however there are many more places in the website where we display a flag and this hover functionality would be beneficial.

**Comment by denthebro** [comment]
> 
> 
> I think hovering over the flag to find out its country is completely reasonable and doesnt take up any precious space on a website. This is currently implemented on the Competition's Results that Alex posted, however there are many more places in the website where we display a flag and this hover functionality would be beneficial.

Which pages do you mean? And I agree, I didn't realize the hover functionality was already implemented on results pages.

**Comment by Jambrose777** [comment]
> > I think hovering over the flag to find out its country is completely reasonable and doesnt take up any precious space on a website. This is currently implemented on the Competition's Results that Alex posted, however there are many more places in the website where we display a flag and this hover functionality would be beneficial.
> 
> Which pages do you mean? And I agree, I didn't realize the hover functionality was already implemented on results pages.

Implemented:
https://www.worldcubeassociation.org/competitions/WuhuOpen2018/results/all?event=333

There are many other pages that have a country flag alone or the country flag + country name which can be cleaned up to the hover functionality.

**Comment by denthebro** [comment]
Example of what Jacob is talking about: https://www.worldcubeassociation.org/results/rankings/333/single

**Comment by denthebro** [comment]
Lack of hover functionality here: https://live.worldcubeassociation.org/
